components/infobox/wikis/ageofempires/infobox_team_custom.lua

```diff
@@ -0,0 +1,61 @@
+---
+-- @Liquipedia
+-- wiki=ageofempires
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Achievements = require('Module:Achievements in infoboxes')
+local Class = require('Module:Class')
+local GameLookup = require('Module:GameLookup')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+
+	-- Automatic achievements
+	team.args.achievements = Achievements.team{team = team.pagename, aka = team.args.aka}
+
+	-- Automatic org people
+	team.args.coach = Template.expandTemplate(frame, 'Coach of')
+	team.args.manager = Template.expandTemplate(frame, 'Manager of')
+	team.args.captain = Template.expandTemplate(frame, 'Captain of')
+
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+
+	_args = team.args
+
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
```

Comment:
Current earnings queries by name, not templatename and therefore player earnings aren't recognised
```suggestion

function CustomInjector:parse(id, widgets)
	if id == 'earningscell' then
		widgets[1].name = 'Approx. Total Winnings'
	end
	return widgets
end

```